worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -94,8 +97,8 @@ function onBeforeProcessDocument(e) {
     }
 
     var args = extractArguments(e.rootDocument);
-
-    e.cancel = ! eval(action.conditionFunction)(e.document, args);
+    eval(action.conditionFunction);
+    e.cancel = ! condition(e.document, args);
```

Comment:
Why are we making this change?  Can't `eval` return a function anymore?

Reply:
No, that's a nashorn specific syntax extension. It can be enabled in graal.js as an experimental feature though.